src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -211,6 +213,17 @@ ParseResult ONNXConstantOp::parse(OpAsmParser &parser, OperationState &result) {
   return success();
 }
 
+// Constant Materializer
+Operation *ONNXDialect::materializeConstant(
+    OpBuilder &builder, Attribute value, Type type, Location loc) {
+
+  // The atrribute could be DenseElemnemntsAttr, IntAttr, FloatAttr and etc.
+  // onnx builder is used to convert it into value()
+  OnnxBuilder onnx(builder, loc);
+  Value result = onnx.constant(value);
+  return result.getDefiningOp();
+}
+
```

Comment:
please move to the end of the file so that all the print and parse methods are next to each other

and please follow the file convention and add`//===-----...` etc to the comment above the method